src/adapters/admixer.js

```diff
@@ -0,0 +1,64 @@
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var Ajax = require('../ajax');
+
+/**
+ * Adapter for requesting bids from Admixer.
+ *
+ * @returns {{callBids: _callBids}}
+ */
+var AdmixerAdapter = function AdmixerAdapter() {
+  var invUrl = '//inv-nets.admixer.net/prebid.aspx';
+
+  function _callBids(data) {
+    var bids = data.bids || [];
+    for (var i = 0, ln = bids.length; i < ln; i++) {
+      var bid = bids[i];
+      var params = {
+        'sizes': bid.sizes.reduce(function (a, b) {
```

Comment:
reduce is failing if i keep size as [320,250]. It only works for [[320,250]].
I would recommend you to use 
`utils.parseSizesInput(bidRequest.sizes);`